src/Kunstmaan/TranslatorBundle/Service/Command/Importer/ImportCommandHandler.php

```diff
@@ -112,6 +119,10 @@ private function importAllBundlesTranslationFiles(ImportCommand $importCommand)
      */
     private function importOwnBundlesTranslationFiles(ImportCommand $importCommand)
     {
+        if (Kernel::VERSION_ID >= 40000) {
```

Comment:
I would put this in the check in the `importBundleTranslationFiles` method and decide there if you want to call `importOwnBundlesTranslationFiles` or `importSf4TranslationFiles`